docker/docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
       # If ENABLE_TELEMETRY is not set, telemetry will start by default
       ENABLE_TELEMETRY: ${ENABLE_TELEMETRY:-true}
       RW_TELEMETRY_TYPE: ${RW_TELEMETRY_TYPE:-"docker-compose"}
-      RW_SECRET_STORE_PRIVATE_KEY_HEX: ${RW_SECRET_STORE_PRIVATE_KEY_HEX:-0123456789abcdef}
+      RW_SECRET_STORE_PRIVATE_KEY_HEX: ${RW_SECRET_STORE_PRIVATE_KEY_HEX:-0123456789abcdef0123456789abcdef}
```

Comment:
Do we have to setup a default `RW_SECRET_STORE_PRIVATE_KEY_HEX` for users? Since we can't alter a private key, if users created a secret with the default private key they'll need to drop all existing secret when they want to switch to their own private key.